### ISSUE TITLE
Update for Tensorflow serving inception_saved_model.py script

### ISSUE THREAD

**Issue by AswathKiruba** [tensorflow/serving#830]
Inception_saved_model.py script under tensorflow/serving/example has the following problems with running the code.

1.The checkpoint file for Incpetion_v3 was not found at /tmp/inception_train/ (no such directory exists). The alternative was to download the  checkpoint(Inception_v3) file available at 
 http://download.tensorflow.org/models/inception_v3_2016_08_28.tar.gz
 2. Secondly on using the checkpoint downloaded from the above link doesn't contain checkpoint proto in it. Hence the `tf.train.get_checkpoint_state()` at line 103 returns None. 

This error could be avoided by editing the code using the tf slim,
`slim = tf.contrib.slim`
`inception = nets.inception`
`with slim.arg_scope(inception.inception_v3_arg_scope()):`
`          logits, _ = inception.inception_v3(images, NUM_CLASSES + 1)
`
And for restoring vaiables,
`variables_to_restore = []`
`for var in tf.contrib.slim.get_model_variables():`
       ` variables_to_restore.append(var)`

`init_fn = tf.contrib.framework.assign_from_checkpoint_fn('./inception_v3.ckpt', variables_to_restore)`
`with tf.Session() as sess:`
         ` init_fn(sess)`

With all these changes above I was able to successfully export the model in tensorflow serving format. I was also able to serve it using docker and get the prediction result. 

Please update the examples, as it would really helpful for the new tensorflow serving users.

**Comment by AswathKiruba** [comment]
We should use the model in this location: http://download.tensorflow.org/models/image/imagenet/inception-v3-2016-03-01.tar.gz
